### PR TITLE
Updating device types for v2 zigbee eui changes and enrollment changes

### PR DESCRIPTION
**Pull request by workingmonk**

### CODE REVIEW

**Comment by workingmonk** [comment]
@tpmanley can you verify the changes done enrollment?
@lbredeso can you confirm the eui update?

**Comment by workingmonk** [comment]
cc @Yaima

**Comment by joneilloine** [comment]
Thank you Vinay!

**Comment by lbredeso** [comment]
@workingmonk The zigbeeEui changes look correct.  The one thing I noticed is that the SmartSense Open/Closed Sensor was missing (noticed mainly because it's the one I've been testing with).  Other than that, I would say :+1: .

**Comment by Yaima** [comment]
:+1:

**Comment by tpmanley** [comment]
I grepped the source for other devicehandlers that still use zigbeeId:

```
grep -RH "0x500" * | grep zigbeeId
nyce-motion-sensor.src/nyce-motion-sensor.groovy:       "zcl global write 0x500 0x10 0xf0 {${zigbeeId}}", "delay 200",
nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy:         "zcl global write 0x500 0x10 0xf0 {${zigbeeId}}", "delay 200",
smartsense-open-closed-accelerometer-sensor.src/smartsense-open-closed-accelerometer-sensor.groovy:     "zcl global write 0x500 0x10 0xf0 {${zigbeeId}}", "delay 200",
smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:     "zcl global write 0x500 0x10 0xf0 {${zigbeeId}}", "delay 200",
tyco-door-window-sensor.src/tyco-door-window-sensor.groovy:     "zcl global write 0x500 0x10 0xf0 {${zigbeeId}}", "delay 200",
```

This includes the open/closed that Luke mentioned.

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy [hunk 1]
```diff
@@ -255,52 +255,54 @@ private Map getMoistureResult(value) {
...
+		"zdo bind 0x${device.deviceNetworkId} 1 ${endpointId} 1 {${device.zigbeeId}} {}", "delay 200",
+		"zcl global write 0x500 0x10 0xf0 {${zigbeeEui}}",
+		"send 0x${device.deviceNetworkId} 1 ${endpointId}", "delay 500",
+		//Enroll Response
+		"raw 0x500 {01 23 00 00 00}",
+		"send 0x${device.deviceNetworkId} 1 1", "delay 200"
     ]
 }
 
+private getEndpointId() {
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy:301]: I may just be missing it, but I don't see this function get called anywhere in this file

Comment (lbredeso) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy:301]: See the calls to endpointId:  Groovy "helpfully" converts these calls to include a camel-cased "get" on the front.

Comment (tpmanley) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy:301]: Ah, got it. Thanks!

**Comment by tpmanley** [comment]
I'm done with the review. Looks good, although I think you should also update the other two smartsense device handlers as part of this PR.

**Comment by workingmonk** [comment]
@tpmanley will update the two smartsense as part of this update. Will work on getting the others done early next week.

**Comment by workingmonk** [comment]
Re-testing complete is all the sensors on v1 and v2.

**Pull request merged**